packagingtest/centos7/systemd/Dockerfile

```diff
@@ -85,6 +80,7 @@ RUN yum -y update; \
     yum -y install systemd; yum clean all;
 
 RUN cd /lib/systemd/system/sysinit.target.wants/; ls -1 | grep -v systemd-tmpfiles-setup.service | xargs rm; \
+    rm -f /lib/systemd/system/multi-user.target.wants/*;\
```

Comment:
Please revert this line change.
See https://github.com/StackStorm/st2packaging-dockerfiles/pull/43/ for context.

Reply:
```suggestion
```

Reply:
FIxed. Also fixed in https://github.com/StackStorm/st2packaging-dockerfiles/blob/master/packagingbuild/Dockerfile.template